api/src/main/java/org/openmrs/module/kenyaemr/reporting/data/converter/definition/evaluator/specialClinics/SpecialClinicsReferredToDataEvaluator.java

```diff
@@ -42,7 +42,7 @@ public EvaluatedEncounterData evaluate(EncounterDataDefinition definition, Evalu
         String qry = "select v.encounter_id,\n" +
                 "(case v.referred_to when 159928 then 'School' when 164407 then 'Other health facility'  when 1000478 then 'CHU' else '' end) as referred_to\n" +
                 "from kenyaemr_etl.etl_special_clinics v\n" +
-                "where date(v.visit_date) between date(:startDate) and date(:endDate) and special_clinic = :specialClinic;";
+                "where date(v.visit_date) between date(:startDate) and date(:endDate) and special_clinic_form_uuid = :specialClinic;";
```

Comment:
Special clinic form uuid is a String. Ensure the parameter value is in quotes, preferrably single quotes.